basilisk/tests/test_accessible_output.py

```
@@ -0,0 +1,539 @@
+            assert handler.speech_stream_buffer == "This has spaces"  # Leading spaces stripped
+
+class TestErrorHandlingAndEdgeCases:
+    """Test error handling and edge cases."""
+
+    def test_init_accessible_output_exception(self):
+        """Test initialization when accessible_output3 raises exception."""
+        with patch('basilisk.accessible_output.config.conf') as mock_conf:
+            mock_conf.return_value.conversation.use_accessible_output = True
+            with patch('accessible_output3.outputs.auto.Auto') as mock_auto:
+                mock_auto.side_effect = Exception("Failed to initialize")
+                with patch('basilisk.accessible_output.log') as mock_log:
```

Ruff (0.11.9)

Local variable `mock_log` is assigned to but never used

Remove assignment to unused variable `mock_log`

(F841)